src/shared/types/learning-object-summary.ts

```diff
@@ -42,6 +53,10 @@ export interface LearningObjectSummary {
    * The collection the Learning Object was submitted to
    */
   collection: string;
+  /**
+   * Summary of Learning object child objects
+   */
+  children?: LearningObjectChildSummary[];
```

Comment:
This property should always be defined. If the object has no children, it will just be defined as an  empty array
```suggestion
  children: LearningObjectChildSummary[];
```

Reply:
Yea, the map function in the driver handles that